README.md

```diff
@@ -1,18 +1,12 @@
-# airtable-clj
```

Comment:
This file doesn't seem like it should've been changed as part of this pull request. Could you revert?

Reply:
This comment still applies.